src/schema/basic_classes.yaml

```diff
@@ -837,12 +837,15 @@ enums:
       novaseq_x:
         aliases:
           - Illumina NovaSeq X
-          - Illumina NovaSeq X Plus 
+          - Illumina NovaSeq X Plus
         comments:
           - Possible flowcell versions are 1.5B, 10B, 25B. Only difference between X and X Plus is 2 flowcells for X Plus versus 1 flowcell for X.
         see_also: 
           - https://www.illumina.com/systems/sequencing-platforms/novaseq-x-plus/specifications.html
-      
+        narrow_mappings:
+          - OBI:0003663
+          - OBI:0003664
```

Comment:
This is "Illumina NovaSeq X Plus". Isn't that an exact mapping?

IF so, I recommend removing the other one.